cmd/crc/cmd/start.go

```diff
@@ -274,6 +274,9 @@ func checkDaemonStarted() error {
 }
 
 func daemonStartedErrorMessage() string {
+	if crcversion.IsMsiBuild() {
+		return "Is CodeReady Containers tray application is running ? Cannot reach daemon API: %v"
```

Comment:
Extra 'is' ("is running" -> "running"), and extra space between "running" and "?", there should be no space

Reply:
fixed!